### ISSUE TITLE
Some tools not generated in config

### ISSUE THREAD

**Issue by Swerving** [asiekierka/TinkeredHegemony#7]
Tools from some mods are not being generated in the config, Even with a bare install and fresh configs with just the mods in question this happens,

-"Immersive Engineering", Steel Tools
-"Thermal Foundation", All Tools
-"Redstone Arsenal". Flux Tools
-"Aether Legacy", All Tools

This happens regardless if Plus TIC or Moar Tinkers is installed or not, The Armors from are hidden if Construct's Armory is installed.

**Comment by asiekierka** [comment]
This might be because they're not strictly *vanilla-style* tools, code-wise. I'd have to take a look.

**Comment by TimGoll** [comment]
Another note: Even adding these items in the config manually does not change anything

**Comment by asiekierka** [comment]
Indeed, the config does not support manual addition of items. :thinking:

**Comment by ilvrak** [comment]
+

**Comment by Burgersim** [comment]
Wanted to ask the same question, everything else seems to work perfectly fine for me, only the steel tools from Immersive Engineering (Axe, Pickaxe and Shovel) are still craftable, the sword is disabled correctly though. It's not the biggest problem, but a little annoyance ^^*

The Aether Legacy Tools are also not disabled, but I stopped using Aether Legacy just thought I'd confirm it,while I can ^^

**Comment by happysmash27** [comment]
I am also having this issue. Has anyone figured out how to fix it?
